app/main.py

```diff
@@ -88,6 +90,41 @@ def _get_client(self):
                                              self.client_id, self.client_secret)
         return KeyVaultClient(credentials)
 
+    def _get_tenant_id(self, tenant_id_from_config):
+        if os.getenv('AUTO_DETECT_AAD_TENANT').lower() != 'true':
```

Comment:
For backward-compatibility, pls change this to 
`if os.getenv('AUTO_DETECT_AAD_TENANT','false').lower() != 'true':`

so that if `AUTO_DETECT_AAD_TENANT` is not set, it will still work.